server/src/main.py

```diff
@@ -179,3 +168,53 @@ async def delete_file(workspace_id: str, path: str):
     # -- for a later date --
     # TODO: Check user has write permission for workspace
     # see @stat()
+
+
+@app.post("/workspaces/{workspace_id}/directory/{directory_path}")
+async def create_directory(workspace_id: str, directory_path: str):
+    empty_data = bytes()
+    data_stream = io.BytesIO(empty_data)
+    client.put_object(
+        workspace_id,
+        directory_path + "/",
+        data=data_stream,
+        length=0,
+        content_type="application/octet-stream",
```

Comment:
Can we try removing this and seeing if it still works. Might be a little cleaner as we don't actually care about the content type of the marker.